### PR TITLE
Added the Rocket Animation with Heading.

### LINKED ISSUE #27: Missing the Rocket animation.
**Actual Behaviour** The app doesn't have the Rocket animation.

Please state here what is currently happening.

**Expected Behaviour** To have the Rocket Animation.

State here what the feature should enable the user to do.

**Steps to reproduce it**

Add steps to reproduce bugs or add information on the place where the feature should be implemented. Add links to a sample deployment or code.

**LogCat for the issue**

Provide logs for the crash here

**Screenshots of the issue**

Where-ever possible add a screenshot of the issue.

**Would you like to work on the issue?**
Yes, I am working on the issue.
Let us know if this issue should be assigned to you or tell us who you think could help to solve this issue.

### PR DESCRIPTION
**Pull request by jddeep**
Fixes #27 and #17 and #29 

**Changes**: I showcased the Rocket Animation.
**Animation 1**
rocketbig
**Animation2**
rocketrepeat
**Screenshot/s for the changes**: @mariobehling I added the .gif to take a look. 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [ ] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [ ] I have requested reviews from other members

**APK for testing**: [Compress the `app-debug.apk` file into a `<feature>.rar` or `<feature>.zip` file and upload it here]

### CODE REVIEW

**Comment by mariobehling** [comment]
Please resolve conflicts. Also take out the word "neurolab" and the icon on the bottom. Would be great if you could add a simple background graphic similar to the original app.

**Comment by jddeep** [comment]
@mariobehling yes i am working on that right now.

**Comment by jddeep** [comment]
@mariobehling can you provide me with the background image you are talking about. Thanks.

**Comment by jddeep** [comment]
Alright sir @mariobehling @CloudyPadmal .

**Comment by jddeep** [comment]
@mariobehling I have removed the "NeuroLab" heading as you wanted. And about the background, should I consider picking an image from here - https://github.com/fossasia/neurolab-desktop/tree/master/src/main/resources . If not kindly direct me to. Thank you.

**Comment by jddeep** [comment]
@CloudyPadmal @mariobehling kindly review and merge the PR. Thanks.

**Comment by CloudyPadmal** [comment]
Refer to [this](https://media.giphy.com/media/9MIlg66ykFVcW2HZBM/giphy-downsized-large.gif) animation.

**Comment by jddeep** [comment]
> Refer to [this](https://media.giphy.com/media/9MIlg66ykFVcW2HZBM/giphy-downsized-large.gif) animation.

Actually I was told to make a Rocket animation which moved up and down. But sir, in this it is more like a continuous animation in which the rocket continues to move forward along the same path @CloudyPadmal

**Comment by CloudyPadmal** [comment]
The background is a continuous animation to let user feel like the rocket is moving forward. With the NeuroLab device, once connected to head, the rocket will move up and down depending on the concentration and stress levels of an individual. 

It is not an easy task to add such a functionality in a single PR. So as a start, here Mario has requested you to add a similar simple brownish hill view background like in the desktop app. The recording is from the desktop app.

**Comment by jddeep** [comment]
> The background is a continuous animation to let user feel like the rocket is moving forward. With the NeuroLab device, once connected to head, the rocket will move up and down depending on the concentration and stress levels of an individual.
> 
> It is not an easy task to add such a functionality in a single PR. So as a start, here Mario has requested you to add a similar simple brownish hill view background like in the desktop app. The recording is from the desktop app.

@CloudyPadmal Okay cool I took a look at the background and will be implementing the same in the app in this PR only.

**Comment by jddeep** [comment]
@CloudyPadmal @mariobehling I just updated the background and animation. Out of the two animation GIFs kindly tell me which one should be implemented. According to that, I will make a final commit here. Thank you.

**Comment by CloudyPadmal** [comment]
Hi, the smaller one looks nice.

**Comment by jddeep** [comment]
Okay @CloudyPadmal will be committing that. Thanks. And do take a look at #30 . Thank you sir.

**Comment by jddeep** [comment]
@CloudyPadmal @mariobehling The PR has been sitting for quite some while now. Kindly look into it. Thank you.

**Comment by mariobehling** [comment]
Ok, thank you. As @CloudyPadmal has pointed out we are looking to convert the desktop app functionalities into the android app. Please refer to the desktop app and set it up on your PC to learn about the next steps. Thank you

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by jddeep** [comment]
@mariobehling conflicts have been taken care of. Now it is good to go. Thank you.

**Comment by jddeep** [comment]
@CloudyPadmal kindly review this PR. Because I wish to make some further updates in successive PRs from my development branch only. Thank you sir.

**Review by CloudyPadmal — APPROVED** [review]
You don't have to wait until one PR gets merged. It will be good to have an understanding about git branches and how to work with them. Thanks.

**Comment by jddeep** [comment]
@CloudyPadmal you can see my past contributions. I have already contributed from 3 branches. Do you suggest opening more? Thanks.

**Comment by CloudyPadmal** [comment]
You can open branches as much as it's necessary. Do some study on git branching. It seems you lack understanding about git version controlling. You don't have to rely on the same branch. Different features can be implemented in different branches. Makes production easier and faster.

**Comment by GOVINDDIXIT** [comment]
> @CloudyPadmal you can see my past contributions. I have already contributed from 3 branches. Do you suggest opening more? Thanks.

It is preferred to work on seperate branch for different issues. It will help you from unnecessary closing your PR.

File: app/build.gradle
Comment (GOVINDDIXIT) [inline app/build.gradle]: Unnecessary. Pls remove

Comment (jddeep) [reply app/build.gradle]: > You can open branches as much as it's necessary. Do some study on git branching. It seems you lack understanding about git version controlling. You don't have to rely on the same branch. Different features can be implemented in different branches. Makes production easier and faster.

@CloudyPadmal Yes, I have been brushing up on my VCS skills lately. And thanks a lot for the suggestion sir.

Comment (jddeep) [reply app/build.gradle]: @GOVINDDIXIT I understand the idea of removing espresso as it is only available for Android 8.0 and above. But runner is necessary for running tests and it facilitates loading the test package and the app under test onto a device or emulator, running your tests, and reporting the results. Moreover these are the basic default dependencies created by Android Studio itself. Is there any concrete reason on why to remove this.

Comment (GOVINDDIXIT) [reply app/build.gradle]: I am asking to remove extra line (line 37)

Comment (jddeep) [reply app/build.gradle]: @GOVINDDIXIT Okay cool sure. I got it.

Comment (jddeep) [reply app/build.gradle]: @GOVINDDIXIT is it a problem if I remove the extra line in the next PR. Because the changes in this PR have already been approved and so I don't want to make any further changes in this one. Is it cool?

Comment (jddeep) [reply app/build.gradle]: @GOVINDDIXIT I removed the extra line in #34 . Thank you.

**Comment by jddeep** [comment]
> You can open branches as much as it's necessary. Do some study on git branching. It seems you lack understanding about git version controlling. You don't have to rely on the same branch. Different features can be implemented in different branches. Makes production easier and faster.

@CloudyPadmal I have been brushing up my VCS skills lately. And thanks for the suggestion.

File: app/build.gradle
Comment (GOVINDDIXIT) [reply app/build.gradle]: > @GOVINDDIXIT I removed the extra line in #34 . Thank you.

This is bad practice. You can always make change in an approved PR, If something got missed

Comment (jddeep) [reply app/build.gradle]: Okay cool will keep that in mind @GOVINDDIXIT . Thanks.

**Comment by jddeep** [comment]
@mariobehling The PR has been approved. It can be merged now. Thank you.

**Comment by jddeep** [comment]
@mariobehling the PR has been approved and can be merged. Thank you,

**Comment by jddeep** [comment]
@mariobehling the PR has been approved. It can be merged now. Thank you.

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]
@jddeep it seems you don't respect the reviews of reviewers. Please do the changes requested by those who review. It doesn't mean the PR is 100% complete if one person approves it. One of the best practices we have is to have a PR reviewed by at-least two reviewers in-case if one misses something. Here I've missed the extra new line which will probably cause merge conflicts if another developer tries to change the gradle file but @GOVINDDIXIT has requested you to change it but you've ignored it. Please make the changes requested by reviewers. These PRs getting too much time and effort because the reviews are simply ignored.

**Comment by jddeep** [comment]
@CloudyPadmal The required change has been seen into. Sir, I did not make the change here but had made the same in the just next PR. I regularly check reviews and make the changes in the same PR. But just for this one, I thought as I was to make just another PR, I would make the change there. I apologise for anything which seemed wrong sir. I had no intention to ignore any review. But anyways, I have made the change here too. Thanks.

**Review by CloudyPadmal — APPROVED** [review]
Nice work. We'll get this merged and move forward. :+1:

**Pull request merged**
